### PR TITLE
snowman: new recipe

### PR DESCRIPTION
**Pull request by return**
Snowman is a cross-platform GUI/CLI application for decompiling native code to C/C++ which supports various architectures (x86/x86-64/ARM) and executable file formats (ELF/Mach-O/PE). Tested on x86_64.

### CODE REVIEW

File: dev-util/snowman/snowman-0.1.0.recipe [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+REVISION="1"
+SOURCE_URI="https://github.com/yegord/snowman/archive/v$portVersion.zip"
+CHECKSUM_SHA256="45a9bf4dd0805c1c42155b6c5ff1c86bb28273dbf1453e958cfeee3cec086bc0"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	snowman$secondaryArchSuffix
+	app:snowman$secondaryArchSuffix
```
Comment (diversys) [inline dev-util/snowman/snowman-0.1.0.recipe]: This should be app:Snowman$secondaryArchSuffix

Comment (pulkomandy) [inline dev-util/snowman/snowman-0.1.0.recipe]: These are (supposedly) case insensitive?

Comment (return) [inline dev-util/snowman/snowman-0.1.0.recipe]: @diversys @pulkomandy  I've ammended the recipe with the suggested changes, are there any other issues with it?

File: dev-util/snowman/snowman-0.1.0.recipe [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	snowman$secondaryArchSuffix
+	app:Snowman$secondaryArchSuffix
+	cmd:nocode$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	libqt4$secondaryArchSuffix
```
Comment (korli) [inline dev-util/snowman/snowman-0.1.0.recipe]: Should be lib:libQt4

File: dev-util/snowman/snowman-0.1.0.recipe [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+	lib:libcapstone$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+	lib:libboost_program_options$secondaryArchSuffix
+	lib:libboost_filesystem$secondaryArchSuffix
+	lib:libboost_iostreams$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	libqt4$secondaryArchSuffix_devel
```
Comment (korli) [inline dev-util/snowman/snowman-0.1.0.recipe]: Should be devel:libQt4

File: dev-util/snowman/snowman-0.1.0.recipe [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+	devel:libcapstone$secondaryArchSuffix
+	devel:libboost_system$secondaryArchSuffix
+	devel:libboost_filesystem$secondaryArchSuffix
+	devel:libboost_iostreams$secondaryArchSuffix
+	devel:libboost_program_options$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:qmake$secondaryArchSuffix
```
Comment (korli) [inline dev-util/snowman/snowman-0.1.0.recipe]: You should indicate which version of qmake you need: cmd:qmake >= 4

**Comment by korli** [comment]
This recipe should probably be located in app-forensics.

**Comment by korli** [comment]
Any reason not to use Qt5?

**Pull request merged**